Akash/html/assign 4/assig2.html

```diff
@@ -24,7 +24,7 @@
     </head>
 
     <body>
-        <h2 style="color: rgb(155, 67, 155);"><center>Jan's Spring Term Afternoon Schedule</center></h2>
+        <h2 style="color: rgb(155, 67, 155);text-align: center;">Jan's Spring Term Afternoon Schedule</h2>
         
         <table style="width:100%;">
             <tr class="a">
```

Comment:
there is no class inside style tag oa name a so if this class is not in use then remove it